src/metadata_validator.py

```diff
@@ -482,7 +482,8 @@ def validate_prop_value(self, prop_name, value, prop_def, msg_prefix):
             
             elif type == "array" or type == "value-list":
                 val = str(value)
-                arr = val.split("*") if "*" in val else val.split(",") if "," in val else [value]
+                list_delimiter = self.model.get_list_delimiter()
+                arr = val.split(list_delimiter) if list_delimiter in val else val.split("*") if "*" in val else [value]
```

Comment:
Since we have default list_delimiter already stored the model, there is no need to try to use "*" to split anymore.

Reply:
Yes for list and value list.  But not sure if array type will also use the same delimiter.

Reply:
Leave '*' delimiter now for array type now. I can update after get confirmed.